### PR TITLE
Add macOS testing docs

### PR DESCRIPTION
**Pull request by rudrankriyam**

### CODE REVIEW

File: content/yaml-testing/testing.md [hunk 1]
```diff
@@ -281,6 +281,25 @@ workflows:
...
+  script: |
+    xcode-project run-tests \
+        --project MyAwesomeApp.xcodeproj \
+        --scheme MyAwesomeApp \
+        --sdk macosx \
+        --output-dir build/macos/test
+test_report: build/macos/test/*.xml
+```
+
+For macOS tests, no destination is specified. Please check [Codemagic CLI tools documentation](https://github.com/codemagic-ci-cd/cli-tools/blob/master/docs/xcode-project/run-tests.md#run-tests) to learn more about more optional arguments to `xcode-project run-tests`.
```
Comment (helinanever) [inline content/yaml-testing/testing.md:301]:
```suggestion
For macOS tests, no destination is specified. Please check [Codemagic CLI tools documentation](https://github.com/codemagic-ci-cd/cli-tools/blob/master/docs/xcode-project/run-tests.md#run-tests) to learn more about optional arguments to `xcode-project run-tests`.
```

File: content/yaml-testing/testing.md [hunk 2]
```diff
@@ -281,6 +281,25 @@ workflows:
...
+
+```bash
+- name: macOS test
+  script: |
+    xcode-project run-tests \
+        --project MyAwesomeApp.xcodeproj \
+        --scheme MyAwesomeApp \
+        --sdk macosx \
+        --output-dir build/macos/test
+test_report: build/macos/test/*.xml
```
Comment (helinanever) [inline content/yaml-testing/testing.md:298]: is the indentation correct?

Comment (nevercode-rudrank) [reply content/yaml-testing/testing.md]: Yikes, updated it

**Review by helinanever — APPROVED** [review]
LGTM

**Pull request merged**